system/ee/ExpressionEngine/Addons/channel/mod.channel.php

```diff
@@ -3200,6 +3211,9 @@ public function category_archive()
                     $chunk = ee()->TMPL->parse_date_variables($chunk, array('entry_date' => $row['entry_date']));
 
                     $row['channel_url'] = parse_config_variables($row['channel_url']);
+                    $row['page_uri'] = $site_pages[$site_id]['uris'][$row['entry_id']];
```

Comment:
I think depending on error reporting level in PHP using this without conditional check might give a notice if the entry does not have corresponding page.

Reply:
And if I tell you this is a plain copy/paste from parser lib? Let's fix it!